v1/src/simulator/src/circuit.ts

```
@@ -66,40 +72,60 @@ export function resetScopeList() {
-    // $(`#${id}`).addClass('current')
-    const index = circuit_list.value.findIndex((circuit) => circuit.id == id); // TODO: add strict equality after typescript
-    circuit_list.value[index].focussed = true;
-    if (activeCircuit.value) {
-      activeCircuit.value.id = globalScope.id;
-      activeCircuit.value.name = globalScope.name;
+    if (!scopeList[id].verilogMetadata.isVerilogCircuit) {
+        verilogModeSet(false)
+        simulatorMobileStore.isVerilog.value = false
+    }
+    // $(`.circuits`).removeClass('current')
+    circuit_list.value.forEach((circuit) =>
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)